capa/features/extractors/elf.py

```diff
@@ -603,6 +604,44 @@ def abi_tag(self) -> Optional[ABITag]:
         return ABITag(os, kmajor, kminor, kpatch)
 
 
+class SYMTAB:
```

Comment:
nit: `SymTab` is probably a better casing for this class, but not a huge deal

Reply:
I agree